File: app/src/main/java/to/mephis/apiscrapercontrol/alarmReceiver.java
package to.mephis.apiscrapercontrol;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class alarmReceiver extends BroadcastReceiver {
    public static final int REQUEST_CODE = 12345;
    @Override
    public void onReceive(Context context, Intent intent) {
        Intent i = new Intent(context, apiPoller.class);
        context.startService(i);
    }
}

File: app/src/main/java/to/mephis/apiscrapercontrol/alarmReceiver.kt
package to.mephis.apiscrapercontrol

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class alarmReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        val i = Intent(context, apiPoller::class.java)
        context.startService(i)
    }

    companion object {
        val REQUEST_CODE = 12345
    }
}
